app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -236,6 +244,12 @@ class CasesService @Inject()(appConfig: AppConfig,
     addEvent(original, updated, details, operator)
   }
 
+  private def addAssignmentChangeEvent(original: Case, updated: Case, operator: Operator, comment: Option[String] = None)
+                                        (implicit hc: HeaderCarrier): Future[Unit] = {
```

Comment:
formatting

Reply:
What's wrong with it?

Reply:
This follows the pattern of every other method in that class

Reply:
the opening bracket of the implicit param is not parallel with the bracket above